Client/gamemanager.cpp

```diff
@@ -9,10 +9,34 @@
 #include "netclient.h"
 #include "settings.h"
 #include <QApplication>
+#include <QDebug>
 #include <QMessageBox>
+#include <QObject>
 
 volatile bool loopEnabled;
 
+/* Handles the Puyolib debug information transfer. See Puyolib/DebugLog.h for more info. */
+void GameManager::handleDebugLog(const std::basic_string<char>& text, ppvs::DebugMessageType severity)
```

Comment:
This should probably be
`const std::basic_string<char>&` -> `const std::string&`

Reply:
Addressed, corrected. A classic case of IDE automatically micro-optimizing.